lib/middleware/xhr.md

```diff
@@ -0,0 +1,86 @@
+### Server -> Client
```

Comment:
documenting cookie stuff while I was learning the little details -- Justin asked me to do it. :)